src/main/java/com/patchworkmc/transformer/PatchworkTransformer.java

```diff
@@ -122,7 +120,7 @@ public void accept(String name, byte[] content) {
 			accessTransformations.addMethodTransformation(subscribeEvent.getMethod(), subscribeEvent.getMethodDescriptor(), AccessTransformation.MAKE_PUBLIC);
```

Comment:
Is this still needed? Making all event handlers public was only needed because of the class generation approach.